<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaylistInfoPage</class>
 <widget class="QWidget" name="PlaylistInfoPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>984</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="1,0,0,0,10">
   <property name="horizontalSpacing">
    <number>24</number>
   </property>
   <property name="verticalSpacing">
    <number>12</number>
   </property>
   <item row="0" column="0" rowspan="4">
    <widget class="QLabel" name="posterLabel">
     <property name="minimumSize">
      <size>
       <width>180</width>
       <height>180</height>
      </size>
     </property>
     <property name="text">
      <string>Poster</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="pageTitleLabel">
     <property name="font">
      <font>
       <pointsize>36</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Brian.Crain</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="pageInfoLabel">
     <property name="text">
      <string>Playlist · 17 songs, 11 hr 9 min 43 sec · Private</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="pageInfo2Label">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="playButton">
       <property name="text">
        <string>Play</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="insertToQueueButton">
       <property name="text">
        <string>Insert to queue</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="appendToQueueEndButton">
       <property name="toolTip">
        <string>Add to the End of Queue</string>
       </property>
       <property name="text">
        <string>Append to queue</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="favorButton">
       <property name="text">
        <string>Favor</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="TrackTableView" name="tableView"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TrackTableView</class>
   <extends>QTableView</extends>
   <header location="global">tracktableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
